charts/tekton-pipelines/Chart.yaml

```diff
@@ -110,6 +110,14 @@ description: |
             script: |
               #!/bin/bash
               echo "hello world2"
+        preDeployTaskSteps:
+          - name: hello1
+            image: node:16
```

Comment:
- let's use some meaningful name not hello
- use latest stable  node version (we tend to copy/paste and therefore let's use fresh stuff)

Reply:
done